pkg/apiclient/apiclient_test.go

```diff
@@ -29,9 +29,10 @@ func TestClient_GetSystemClient(t *testing.T) {
 			func() (*octopusApiClient.Client, error) {
 				return factory.GetSystemClient(&apiclient.FakeRequesterContext{})
 			})
-		//clientReceiver2 := testutil.GoBegin2(func () (*octopusApiClient.Client, error) { factory.GetSystemClient(&apiclient.FakeRequesterContext{})
 
+		//clientReceiver2 := testutil.GoBegin2(func () (*octopusApiClient.Client, error) { factory.GetSystemClient(&apiclient.FakeRequesterContext{})
 		api.ExpectRequest(t, "GET", "/api").RespondWith(root)
+		api.ExpectRequest(t, "GET", "/api/spaces").RespondWith(root)
```

Comment:
all of the calls to `/api/spaces` was due to a previous update to the go-octopusdeploy client library